bin/wasm-node/javascript/src/webrtc/index.ts

```diff
@@ -210,20 +210,30 @@ export default function(targetIp: string, protocol: 'tcp' | 'udp', targetPort: n
             // The maximum SCTP user message size (in bytes) (RFC8841)
             "a=max-message-size:100000" + "\n" +
             // A transport address for a candidate that can be used for connectivity checks (RFC8839).
-            "a=candidate:0 1 " + (protocol == 'tcp' ? "TCP" : "UDP") + " 2113667327 " + targetIp + " " + targetPort + " typ host" + "\n";
+            "a=candidate:1 1 " + (protocol == 'tcp' ? "TCP" : "UDP") + " 2113667327 " + targetIp + " " + targetPort + " typ host" + "\n" +
+            // Indicate that there will be no further candidates.
+            "a=end-of-candidates" + "\n";
```

Comment:
From what I remember, I believe that technically you need to change the `a=ice-options:` attribute to the version that supports this

Reply:
right, it seems that it requires `ice-options:trickle`. removed